compiler/src/dmd/frontend.h

```diff
@@ -2891,7 +2891,7 @@ enum : int32_t { TEST_EMIT_ALL = 0 };
 
 extern void genhdrfile(Module* m, OutBuffer& buf);
 
-extern void moduleToBuffer(OutBuffer* buf, Module* m);
+extern void moduleToBuffer(OutBuffer& buf, Module* m);
```

Comment:
I'd expect there to be another copy of this function in the hand-maintained headers used by gdc/ldc.

Reply:
There is. Fixed.